source/PreferencesPanel.cpp

```diff
@@ -117,6 +117,8 @@ bool PreferencesPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &comma
 		Exit();
 	else if(key == 'c' || key == 's' || key == 'p')
 		page = key;
+	else if(key == 'o' && page == 'p')
+		SDL_OpenURL(("file://" + Files::Config() + "plugins").c_str());
```

Comment:
The Files class is generally where we want all the platform-dependent handling. So this should be a call to a method there, e.g. `Files::OpenFolder` that takes an enum (plugins, saves currently).

The implementation can then feature test SDL for this, and provide OS-specific fallbacks. (Having one place for the callout to SDL also means we don't have multiple or missing checks for SDL errors.)